src/main/resources/scripts/kill counter.sk

```diff
@@ -2,25 +2,24 @@
 # Counts kills per life, in total, and the highest kill streak.
 #
 
-# Defaults are useful here, as a player's kills would be '<none>' if he hasn't killed anything yet,
-# but with defaults defined those will be used in that case.
-variables:
-	{kill counter.%player%.kills_session} = 0
-	{kill counter.%player%.kills_session_max} = 0
-	{kill counter.%player%.kills_total} = 0
-
 on death:
-	attacker is a player
-	add 1 to {kill counter.%attacker%.kills_total}
-	add 1 to {kill counter.%attacker%.kills_session}
-	{kill counter.%attacker%.kills_session} is greater than {kill counter.%attacker%.kills_session_max}:
-		set {kill counter.%attacker%.kills_session_max} to {kill counter.%attacker%.kills_session}
+	if {killcounter::%attacker%::*} doesn't exist:
+		set {killcounter::%attacker%::kills-this-life} to 0
+		set {killcounter::%attacker%::kill-streak} to 0
+		set {killcounter::%attacker%::kills} to 0
+	if attacker is a player:
+		add 1 to {killcounter::%attacker%::kills}
+		add 1 to {killcounter::%attacker%::kills-this-life}
+		if {killcounter::%attacker%::kills-this-life} is greater than {killcounter::%attacker%::kill-streak}:
+			set {killcounter::%attacker%::kill-streak} to {killcounter::%attacker%::kills-this-life}
 
 on death of player:
-	set {kill counter.%player%.kills_session} to 0
```

Comment:
Using spaces in variable names is encouraged if it looks cleaner that way and/or when it is grammatically correct.

Reply:
Okay, I'll fix it.